src/commands/general/stats.ts

```diff
@@ -25,15 +25,17 @@ export default class extends Command {
 
         const uptime = convertTime(message, this.client.uptime ?? 0);
         if (!message.embeddable)
-            return message.send(message.translate('general/stats:TEXT', {
-                uptime,
-                guilds: guilds.toLocaleString(),
-                count: this.client.shardCount,
-                users: users.toLocaleString(),
-                cpu: Math.round(loadavg()[0] * 10000) / 100,
-                usedRAM,
-                totalRAM
-            }));
+            return message.send([
+                message.translate('general/stats:TEXT_1'),
+                '```autohotkey',
+                message.translate('general/stats:TEXT_2', { uptime }),
```

Comment:
I would recommend changing these names since each is separate now and not a union of multiple strings.  For example this could be `'general/stats:UPTIME'`

Reply:
There is a difference between `general/stats:TEXT_2` and `general/stats:UPTIME`, thus I will not change it.